packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -0,0 +1,31 @@
+/* global before, browser, Terra */
+
+describe('Overlay', () => {
+  describe('Light Themed', () => {
+    before(() => browser.url('/#/tests/overlay-tests/light-themed'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+    Terra.should.themeEachCustomProperty({
+      '--terra-overlay-content-color': 'red',
+      '--terra-overlay-content-padding': '1000px',
+      '--terra-overlay-light-background-color': 'green',
+      '--terra-overlay-light-background-image': 'linear-gradient(red, green)',
+      '--terra-overlay-light-opacity': '0.8',
+    });
+  });
+
+  describe('Dark Themed', () => {
```

Comment:
We should add screenshots of clear theme.

Reply:
Yes I added it after the dark theme tests.